Samples/TardisBank/Solutions/Suteki.TardisBank.Web.Mvc/Global.asax.cs

```diff
@@ -75,6 +62,18 @@ protected void Application_Start()
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
 
+        ISessionFactory CreateSessionFactory(IKernel kernel)
```

Comment:
@shatl not sure how multi tenancy would affect the location of this code (I think multi tenancy is way far down the line, unless you meant multi database which would be sooner), but we can still add whatever functionality we want to the Installer instead of having this code here until then.